solr/core/src/test/org/apache/solr/cli/ZkSubcommandsTest.java

```diff
@@ -507,9 +507,10 @@ public void testGetFile() throws Exception {
     Path file =
         tmpDir.resolve("solrtest-getfile-" + this.getClass().getName() + "-" + System.nanoTime());
 
+    // Not setting --zk-host, will fall back to sysProp 'zkHost'
     String[] args =
         new String[] {
-          "cp", "-z", zkServer.getZkAddress(), "zk:" + getNode, file.toAbsolutePath().toString()
+          "cp", "zk:" + getNode, file.toAbsolutePath().toString()
```

Comment:
Simple way of testing the fix. This test explicitly sets `zkHost` sysProp in setUp, so removing `-z` shuold cause a fallback. The test `ZkSubcommandsTest#testGetFile()` will thus fail until the fix is added.